### PR TITLE
[ADD][purchase_update_seller_price] - Module to update the seller pri…

### PR DESCRIPTION
**Pull request by ougc27**
Hi @alan196 could you review this PR please?

Thank you.
Regards

### CODE REVIEW

File: purchase_update_seller_price/README.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+
+=================================
+Purchase Update Seller PRice
+=================================
+
+This module add the posibility to update the seller price when a PO is confirmed.
```
Comment (alan196) [inline purchase_update_seller_price/README.rst]: This module add the posibility to update the seller price every time a PO is confirmed.

File: purchase_update_seller_price/README.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+Usage
+=====
+
```
Comment (alan196) [inline purchase_update_seller_price/README.rst]: This module don't need configuration.

File: purchase_update_seller_price/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+{
+    "name": "Purchase Update Seller Price",
+    "summary": "Update the seller price on the product",
+    "version": "11.0.1.0.0",
+    "license": "LGPL-3",
+    "website": "https://github.com/purchase-workflow",
+    "author": [
+        "JARSA Sistemas S.A. de C.V.",
+        "Odoo Community Association (OCA)"
```
Comment (alan196) [inline purchase_update_seller_price/__manifest__.py]: , at the end

File: purchase_update_seller_price/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2018 JARSA Sistemas S.A. de C.V.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from . import purchase
```
Comment (alan196) [inline purchase_update_seller_price/models/__init__.py]: from . import purchase_order... change file name

**Review by alan196 — CHANGES_REQUESTED** [review]

**Comment by ougc27** [comment]
@alan196 the changes are done!

**Review by alan196 — APPROVED** [review]

**Pull request merged**